core/services/log/registrations.go

```diff
@@ -210,6 +210,14 @@ func (r *registrations) sendLog(log types.Log, orm ORM, latestHead models.Head,
 			continue
 		}
 
+		if !latestHead.IsInChain(log.BlockHash) {
+			logger.Tracew("Skipping because not in canonical ",
```

Comment:
should it say "not in canonical longest chain" ?

Reply:
removed this part for now